bin/lib/zopen-build

```diff
@@ -376,6 +362,25 @@ checkEnv()
     fi 
   fi
 
+  # Decide which compiler is being used XL or CLANG
+  if [ "${ZOPEN_COMP}x" = "x" ]; then
+    export ZOPEN_COMP="XL" # Change the default to clang
+  else
+    # user specified, so normalize in upper case
+    ZOPEN_COMP=$(echo "${ZOPEN_COMP}" | tr '[a-z]' '[A-Z]')
+  fi
+
+  echo "IN ${ZOPEN_COMP}"
+  case "${ZOPEN_COMP}" in 
+    XL)
+      implicitDeps="$implicitDeps ibm_xlclang"  
+    ;;
+    CLANG)
+      implicitDeps="$implicitDeps ibm_clang"  
```

Comment:
Just call it clang.  We also have the ibm-clang command from OpenXL.  Calling the OEL clang ibm_clang will just confuse things and would make adding ibm-clang support interesting (would it be ibm_ibm-clang :) ). 

Maybe we should change  the --comp option so it specifies the name of the dependency.  This would avoid this mapping step and would allow us to add support  for additional compilers without needing to modify this script.  I'd name all the compiler dependencies comp_XXX.  That way we can easily list all compilers supported in the help and at this point add `comp_${ZOPEN_COMP}` to the implicit dependencies.

Reply:
Good suggestions!